src/notification-hub/azext_notification_hub/commands.py

```diff
@@ -7,59 +7,32 @@
-        g.custom_show_command('show', 'get_authorization_rule_notificationhubs_namespace')
-        g.custom_command('list', 'list_authorization_rules_notificationhubs_namespace')
-        g.custom_command('create', 'create_or_update_authorization_rule_notificationhubs_namespace')
-        g.custom_command('delete', 'delete_authorization_rule_notificationhubs_namespace', confirmation=True)
-        g.custom_command('list-keys', 'list_keys_notificationhubs_namespace')
-        g.custom_command('regenerate-keys', 'regenerate_keys_notificationhubs_namespace')
-
-    from ._client_factory import cf_notification_hubs
-    notificationhubs_notification_hubs = CliCommandType(
-        operations_tmpl='azext_notification_hub.vendored_sdks.notificationhubs.operations.notification_hubs_operations#NotificationHubsOperations.{}',
-        client_factory=cf_notification_hubs)
-    with self.command_group('notification-hub', notificationhubs_notification_hubs, client_factory=cf_notification_hubs, is_experimental=True) as g:
-        g.custom_command('create', 'create_notificationhubs_hub')
-        g.custom_command('update', 'update_notificationhubs_hub')
-        g.custom_command('delete', 'delete_notificationhubs_hub', confirmation=True)
-        g.custom_show_command('show', 'get_notificationhubs_hub')
-        g.custom_command('list', 'list_notificationhubs_hub')
-        g.custom_command('check-availability', 'check_notification_hub_availability_notificationhubs_hub')
-        g.custom_command('test-send', 'debug_send_notificationhubs_hub', validator=validate_notification_message)
-
-    with self.command_group('notification-hub authorization-rule', notificationhubs_notification_hubs, client_factory=cf_notification_hubs) as g:
-        g.custom_command('list', 'list_authorization_rules_notificationhubs_hub')
-        g.custom_show_command('show', 'get_authorization_rule_notificationhubs_hub')
-        g.custom_command('create', 'create_or_update_authorization_rule_notificationhubs_hub')
-        g.custom_command('delete', 'delete_authorization_rule_notificationhubs_hub')
-        g.custom_command('list-keys', 'list_keys_notificationhubs_hub')
-        g.custom_command('regenerate-keys', 'regenerate_keys_notificationhubs_hub')
-
-    with self.command_group('notification-hub credential', notificationhubs_notification_hubs, client_factory=cf_notification_hubs) as g:
-        g.custom_command('list', 'get_pns_credentials_notificationhubs_hub')
-        g.custom_command('gcm update', 'update_gcm_credential')
-        g.custom_command('apns update', 'update_apns_credential')
-        g.custom_command('wns update', 'update_wns_credential')
-        g.custom_command('mpns update', 'update_mpns_credential')
-        g.custom_command('adm update', 'update_adm_credential')
-        g.custom_command('baidu update', 'update_baidu_credential')
```

Comment:
According the the definition, there's not create command, but only update command.